splice_machine/src/main/java/com/splicemachine/derby/ddl/DDLUtils.java

```diff
@@ -309,6 +309,14 @@ public static BitSet getIndexedCols(int[] indexColsToMainColMap) {
         }
     }
 
+    public static void preDropForeignKey(DDLMessage.DDLChange change, DataDictionary dd) throws StandardException {
```

Comment:
Nitpick, name is a bit confusing since it's being used for both Add/Drop, preAlter.. might be better.

Reply:
Thanks Daniel, you're right, I will address it in another PR where I improve the FK checker.